tests/conftest.py

```diff
@@ -820,7 +253,7 @@ def archive():
         # To tear down, we need to blow away the archive server's directory.
         # For the test suite, we've also mounted that directory locally, so
         # we can do that
-        archivebase = f"{os.getenv('SEECHANGE_TEST_ARCHIVE_DIR')}/{cfg.value('archive.path_base')}"
+        archivebase = f"{test_config.value('archive.local_read_dir')}/{test_config.value('archive.path_base')}"
```

Comment:
See my comment on seechange_config_test.yaml -- I think it's hazardous to reuse the archive configuration options for something that the tests need internally, but that's not really correct for the way the archive itself works.

Reply:
Don't we want to have a different folder for the archive when running tests? What's the difference if I put it into the config vs. if I use an environmental variable?